src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorWorkflowStep.groovy

```diff
@@ -37,7 +37,11 @@ class IqPolicyEvaluatorWorkflowStep
 {
   String iqStage
 
-  String iqApplication
+  String applicationSelectTypePost
+
+  String listAppId
```

Comment:
These names could use some consideration to be more obvious to the pipeline DSL. Willing to hear suggestions and will think on it myself.